app/src/utils/env.ts

```diff
@@ -0,0 +1,22 @@
+export function get(key: string, def = ''): string {
+    return process.env[key]?.trim() ?? def;
+}
+
+export function getBool(key: string, def = false): boolean {
+    const value = process.env[key];
+
+    if (value)
+        switch (value.trim().toUpperCase()) {
+            case 'TRUE':
+                return true;
+            case 'FALSE':
+                return false;
+        }
+
+    return def;
+}
+
+export function getInt(key: string, def: number): number {
+    const value = process.env[key];
+    return value ? parseInt(value) : def;
```

Comment:
Strictly speaking I pulled this off one of @microwavekonijn's repos, what would you suggest for a fix?

Reply:
Best thing would be to have somewhere a function similar to what's shown in the reference (called filterInt). Although I'd recommend that it return the type `number | null` or `number | undefined` for better type checking, instead of returning `NaN` on error which is just a subtype of `number`.

Reply:
I just added that one for inclusion, never really looked at it critically.

Reply:
Since we're in full control of the codebase, and if people put a string in as a number, parseInt will still parse it, I wouldn't say this requires too much attention.